docs/discovery.md

```diff
@@ -212,7 +212,7 @@ You can contribute a new discovery backend to Swarm. For information on how to d
 
 ## Docker Swarm documentation index
 
-- [User guide](index.md)
-- [Sheduler strategies](/scheduler/strategy.md)
-- [Sheduler filters](/scheduler/filter.md)
-- [Swarm API](/api/swarm-api.md)
+- [User guide](https://docs.docker.com/swarm/)
```

Comment:
Thanks for the feedback, @moxiegirl!

I think what happened was I was reading the docs on Github rather than docs.docker.com. I just took another look, and the [User Guide](https://github.com/docker/swarm/blob/master/docs/index.md) is the only of the four links that works on https://github.com/docker/swarm/blob/master/docs/discovery.md,, but the only link that _doesn't_ resolve correctly on https://docs.docker.com/swarm/discovery/.  

It'd be great if the links worked on both Github and docs.docker.com, but if the latter is the supported source, I'm okay stripping this PR to just fix the spelling typo itself. Thanks again!